bigtable/bigtable.go

```diff
@@ -81,6 +83,13 @@ func NewClientWithConfig(ctx context.Context, project, instance string, config C
 		// TODO(grpc/grpc-go#1388) using connection pool without WithBlock
 		// can cause RPCs to fail randomly. We can delete this after the issue is fixed.
 		option.WithGRPCDialOption(grpc.WithBlock()))
+
+	// Expose an experimental flag to make client attempt DirectPath.
+	// Once e2e tests are finished, this flag will be removed, and client
```

Comment:
What does "Once e2e tests are finished" mean? Seems like we should be able to add this option from the test directly, rather than through an environment variable?

As-is, this seems to be doing more than adding a test, given this is a non-test file. In general, we should avoid adding to the public API when we know we're going to remove it in the future.

Reply:
We are going to release directpath for bigtable, and we designed some e2e test scenarios that needs to be covered before the release (See the background and what we have done for java-bigtable in go/directpath-e2e-test-scenarios). I replace the environment with a global variable, but the idea should be the same.

We need this option now because we do not want to rashly set `internaloption.EnableDirectPath` to true, which is what we will do after all e2e tests. A similar PR for previous work in java-bigtable can be found https://github.com/googleapis/java-bigtable/pull/467

Reply:
Sorry, I'm still confused. Can we avoid adding anything to the exported API? Can the test pass the `internaloption.EnableDirectPath` option when appropriate?

Reply:
Hey Tyler, sorry I did not make myself clear. So DirectPath is a new feature for bigtable, and ultimately we will need to add `internaloption.EnableDirectPath(true)` to the library code to support this feature. For now, we are using a flag that is passed from tests in `internaloption.EnableDirectPath()` because we needs to test this feature first. But since ultimately we will change the library code, I think it is reasonable to add the internal option now. Does this make sense to you? Please let me know if you have more questions. Thanks!

Reply:
I moved the internal option to test files, and the flag is removed.